modules/mono/mono_gd/gd_mono_class.h

```diff
@@ -59,13 +59,12 @@ class GDMonoClass {
 
 		MethodKey() {}
 
-		MethodKey(const StringName &p_name, int p_params_count) {
-			name = p_name;
-			params_count = p_params_count;
+		MethodKey(const StringName &p_name, uint16_t p_params_count) :
+				name(p_name), params_count(p_params_count) {
 		}
 
 		StringName name;
-		int params_count;
+		uint16_t params_count;
```

Comment:
It would be good to give it a default value here.

Reply:
It's always initialized in the constructor, in the call to update the method signature. Is there a performance penalty in adding a default value here?

Reply:
Done!